src/Integration/Binding/IBindingProcess.cs

```diff
@@ -34,8 +34,6 @@ namespace SonarLint.VisualStudio.Integration.Binding
     /// making the changes to projects and files.</remarks>
     internal interface IBindingProcess
     {
-        bool PromptSaveSolutionIfDirty();
```

Comment:
I delete the two solution-save related steps from the interface, and the associated implementation and tests.